src/components/VoiceSearch/VoiceSearch.js

```diff
@@ -94,7 +94,6 @@ class VoiceSearch extends Component {
     this.recognition.onerror = e => {
       this.setState({ errorCode: e.error });
     };
-    // this.recognition.onnomatch = e => console.log('onnomatch', e);
```

Comment:
This is unrelated; prefer not mixing several intents in a same patch (here: remove the comment in another patch, or just search back in the history of your not-yet-merged commits to not introduce comments in the first place)